site/topics/testing/unittest.rst

```diff
@@ -45,4 +45,5 @@ Running Unit Tests
 For Next Class
 ==============
 
+* Check out the test folder in the `GitHub repo <https://github.com/jameshughes89/cs101>`_  to see the unit tests written for the course content 
```

Comment:
Maybe worth linking directly to the tests directory? Unless the intention is also to say "contribute, dammit."

Reply:
It was originally a link right to the tests, but figured, _eh, maybe they'll poke around a little, which would be good_.